File: C#/chap02/4.cs
using System;

namespace Wrox.ProCSharp.Basics
{
   public class ScopeTest
   {
      public static int Main(string[] args)
      {
         int j = 10;
         for(int i = 0; i < 10; ++i)
         {
            int j = 30; // Allowed in C++ (name hiding) but not in C#
            Console.WriteLine(j + i);
         }
         return 0;
      }
   }
}

File: C#/chap02/4.cpp
#include <iostream>
using namespace std;

int main()
{
   int j = 20;
   for(int i = 0; i < 10; ++i)
   {
      int j = 80; // Allowed in C++ but not in C#. Read more about C++ data hiding.
      cout<<(j+i)<<endl;
   }
   cout<<j<<endl;
   return 0;
}

